### PR TITLE
Feedback based on implementing responses in ICARE platform

### PR DESCRIPTION
**Pull request by Dtphelan1**
Small changes that clarified some questions I had when implementing the new response format on the ICARE-platform repo. Also moved the index.md to README.md for convenient rendering on GitHub. Let me know if I should change the file back.

### CODE REVIEW

**Review by rdingwell — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ When the ICAREdata Study submission infrastructure accepts and processes a data
 
 When the ICAREdata Study submission infrastructure rejects a data submission request, the MessageHeader will have the response code value, "fatal-error" and reference an [OperationOutcome](http://hl7.org/fhir/DSTU2/operationoutcome.html) resource contained in another entry in the bundle.
```
Comment (Dtphelan1) [inline README.md]:
```suggestion
When the ICAREdata Study submission infrastructure rejects a data submission request, the MessageHeader will have the response code value, "fatal-error" and reference an [OperationOutcome](https://www.hl7.org/fhir/operationoutcome.html) resource contained in another entry in the bundle.
```

**Review by jafeltra — APPROVED** [review]
Looks great!

File: README.md [hunk 2]
```diff
@@ -132,7 +132,7 @@ The OperationOutcome will have issue severity value, "error" and an issue code v
       - OperationOutcome
         - id -- _identifier of this OperationOutcome resource; referenced above in MessageHeader_
         - issue
-          - severity -- "fatal-error" _The literal value, "fatal-error" denotes the ICAREdata system cannot recover from the reported error and will not process the request._
+          - severity -- "fatal | error" -- _See [Operation Outcome](https://www.hl7.org/fhir/operationoutcome-examples.html) in the FHIR Specification_
```
Comment (jafeltra) [inline README.md]: Should this link also be https://www.hl7.org/fhir/operationoutcome.html?

**Comment by jafeltra** [comment]
Re-approve!

**Pull request merged**